libraries/standard/mqtt/src/mqtt.c

```diff
@@ -630,18 +690,22 @@ static MQTTStatus_t handleIncomingPublish( MQTTContext_t * const pContext,
 /*-----------------------------------------------------------*/
 
 static MQTTStatus_t handleIncomingAck( MQTTContext_t * const pContext,
-                                       MQTTPacketInfo_t * pIncomingPacket )
+                                       MQTTPacketInfo_t * pIncomingPacket,
+                                       bool manageKeepAlive )
 {
     MQTTStatus_t status = MQTTBadResponse;
     MQTTPublishState_t publishRecordState = MQTTStateNull;
     uint16_t packetIdentifier;
     /* Need a dummy variable for MQTT_DeserializeAck(). */
-    bool sessionPresent = false;
+    bool sessionPresent = false, callAppCallback = true;
```

Comment:
(Minor) The declaration of `callAppCallback` seems to be misplaced in relation to the comment about `MQTT_DeserializeAck`

Reply:
Was handled in PR #990